contracts/cw3-fixed-multisig/src/contract.rs

```diff
@@ -809,7 +811,17 @@ mod tests {
             }
         );
 
-        // TODO: non-Open proposals cannot be voted
+        // non-Open proposals cannot be voted
+        // Vote it again
+        let env = mock_env(VOTER5, &[]);
+        let res = handle(&mut deps, env, yes_vote);
```

Comment:
After adding this test, I was left wondering that maybe is a good idea to allow voting on `Passed` proposals (L165](https://github.com/CosmWasm/cosmwasm-plus/blob/f979892bb7d32339769ad83ccb289e92e5a2f300/contracts/cw3-fixed-multisig/src/contract.rs#L165) above).

That way, we can get better statistics on the final result of a given proposal. That is, despite a proposal having passed, it can have a large number of No / Veto votes (some of which can be cast after it passed). Or, conversely, a proposal can have a large adoption, than the one reflected by the results, which currently "freeze" after it passed.


That way you, can get better statistics